engine/datastore/where_mocks_test.go

```diff
@@ -47,7 +47,7 @@ func mockClient(engine Engine) (*Client, *gorm.DB) {
 	client, _ := clientInterface.(*Client)
 	client.options.engine = engine
 	dialector := mockDialector(engine)
-	gdb, _ := gorm.Open(dialector, &gorm.Config{})
+	gdb, _ := gorm.Open(dialector, &gorm.Config{TranslateError: true})
```

Comment:
That's the general comment not related strictly to the PR. I see that for some reason there is no error checking in funcs: _mockDialector, mockClient_. I would suggest creating a separate task and marking them as test helpers.

https://google.github.io/styleguide/go/best-practices#leave-testing-to-the-test-function

Reply:
I talked to @dorzepowski about that, and generally we don't see a need to introduce it. Especially now where there are many more important tasks to do.

Reply:
@mgosek-4chain  About the `t.Helper`

I talked to @dorzepowski about that, and generally we don't see a need to introduce it. Especially now where there are many more important tasks to do.